easybuild/easyconfigs/i/icmake/icmake-7.23.02-foss-2016a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PackedBinary'
+
+name = 'icmake'
+version = '7.23.02'
+
+homepage = 'http://icmake.sourceforge.net/'
+description = """Icmake is a hybrid between a 'make' utility and a 'shell script' language. Originally, it was written to
+ provide a useful tool for automatic program maintenance and system administrative tasks on old MS-DOS platforms."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/fbb-git/%(name)s/archive/']
+sources = ['%(version)s.tar.gz']
+
+install_cmd = 'cd %(name)s-%(version)s/%(name)s && sed -i \'s|usr/||g\' INSTALL.im && ./icm_bootstrap %(installdir)s && ./icm_install strip all /'
+
+sanity_check_paths = {
+                      'files': ['bin/icmake', 'bin/icmbuild', 'bin/icmstart', 'bin/icmun',
+                                'lib/icmake/icm-comp', 'lib/icmake/icm-exec', 'lib/icmake/icm-pp'],
+                      'dirs': []
+                     }
```

Comment:
@RvDijk please fix indent (indent `'files': ...` & `'dirs`: ...` with just 4 spaces, and align the list entries if line wrapping is still required like you're doing here; last `}` should be left-aligned)